llpc/test/shaderdb/hlsl/PipelineHlsl_TestRuntimeArrayPacking.pipe

```diff
@@ -0,0 +1,430 @@
+; Check that the corner case that HLSL-derived shader that has a variable-sized array with struct-type element whose last member is variable-sized array.
+; The case can work well by not padding the outmost variable-sized array.
+
+; BEGIN_SHADERTEST
+; RUN: amdllpc -spvgen-dir=%spvgendir% -v %gfxip %s | FileCheck -check-prefix=SHADERTEST %s
+; SHADERTEST: AMDLLPC SUCCESS
```

Comment:
Is it possible to add more pattern check in this test? Just check AMDLLPC SUCCESS is too simple. I prefer to add some IR detecting after translator or lower passes finish and output the text results.

Reply:
Sure. I add a pattern by checking whether the global variable is padded.